### PR TITLE
Adds Chicken Tenders

### PR DESCRIPTION
**Pull request by Windos-7**
## About the PR 
You can now cut chicken nuggets into 3 strips, which you can then process into chicken tenders. Tendies have randomized names. You can even dip chicken tendies in bowls like tortillas!
Demonstration video
https://files.catbox.moe/9hnhrv.mp4

## Why's this needed? 
tendies

## Changelog

```changelog
(u)Fwll
(*)Added dippable chicken tenders
```

### CODE REVIEW

**Comment by Eagletanker** [comment]
Uhhh
Nuggets are smaller than tenders, so it should be chicken tenders to nuggets 
Small nitpick

File: code/modules/food_and_drink/ingredients.dm [hunk 1]
```diff
@@ -136,6 +136,24 @@
...
 			icon_state = "nugget1"
 		return ..()
+	attackby(obj/item/W as obj, mob/user as mob)
+		if (iscuttingtool(W))
+			var/turf/T = get_turf(src)
+			user.visible_message("[user] cuts [src] into slices.", "You peel [src] and cut it into strips.")
+			var/makeslices = 3
+			while (makeslices > 0)
+				new /obj/item/reagent_containers/food/snacks/ingredient/meat/chickenstrip(T)
+				makeslices -= 1
```
Comment (TobleroneSwordfish) [inline code/modules/food_and_drink/ingredients.dm:146]:
```suggestion
			for (var/i in 1 to 3)
				new /obj/item/reagent_containers/food/snacks/ingredient/meat/chickenstrip(T)
```

**Comment by NightmareChamillian** [comment]
can't wait to spend the entire round doing packet wizardry in tech storage whilst munching on tendies

**Comment by Kyle2143** [comment]
Remove the references to the tendie meme.

**Review by Kyle2143 — CHANGES_REQUESTED** [review]
above

**Pull request closed without merging**